### ISSUE TITLE
Training and test set have overlapping moments

### ISSUE THREAD

**Issue by twoertwein** [sunlightsgy/MEmoR#1]
Thank you for creating and sharing this dataset!

I was surprised to see that the training/test split in your paper have overlapping moments. For example, the moment 95.354484s from session S09E24_001 is for Leonard in the training set but for Penny in the test set. This can be problematic as characters are probably more likely to express similar emotions during the same moment. Out of the 4562 moments, 1007 overlap between the training and test set.
